### PR TITLE
Support service name in global header

### PR DESCRIPTION
**Pull request by teneightfive**
This adds support for a service to display its name next to the logo
in the global header.

### CODE REVIEW

File: src/templates/nhsuk-base.njk [hunk 1]
```diff
@@ -48,6 +48,8 @@
               <a href="/" class="global-header__link">
                 <img src="{{ asset_path('assets/images/logotype-nhs-mono.png') }}" alt="NHS">
               </a>
+              {% block service_name %}
```
Comment (teneightfive) [inline src/templates/nhsuk-base.njk]: @tyom not sure the best route here. We can either support an empty block and the project using this package can add `<span class="global-header__service-name">Service name</span>` but we'd need to document this markup. 

Or we can use a variable like we are for title, and check it exists before rendering the markup. So would be more like:
```
{% if serviceName %}
  <span class="global-header__service-name">{{ serviceName }}</span>
{% endif %}
```

We could also combine the two:
```
{% block service_name %}
  {% if serviceName %}
    <span class="global-header__service-name">{{ serviceName }}</span>
  {% endif %}
{% endblock %}
```

Comment (tyom) [reply src/templates/nhsuk-base.njk]: I think I prefer the combined approach as it gives the flexibility to customise it further if needs be.

Comment (teneightfive) [reply src/templates/nhsuk-base.njk]: What's our preferred pattern for variable names? Snakecase or camelcase?

We're using snakecase for block names.

Comment (tyom) [reply src/templates/nhsuk-base.njk]: camelCase I reckon.

Comment (teneightfive) [reply src/templates/nhsuk-base.njk]: Cool, check the fixup and I'll rebase

File: README.md [hunk 2]
```diff
@@ -102,6 +102,7 @@ that can be overridden or extended from your template:
   - `body_skiplinks` - wraps container with "skip to content" link (first element inside `body`)
   - `body_notifications` - wraps cookie message container (above `site_header`, after `skiplinks`)
   - `body_site_header` - wraps site header
+  - `header_service_title` - wraps the service title
```
Comment (tyom) [inline README.md]: Maybe move it under `head`?

**Review by tyom — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (teneightfive) [reply README.md]: But it's not part of the`head`. It's part of the global body header?

Comment (tyom) [reply README.md]: Sorry, you're right. Although could add the container name to it's name to be more consistent with the convention. i.e. `body_site_header_service_title`.

Comment (teneightfive) [reply README.md]: Done and rebased.

**Review by tyom — APPROVED** [review]

**Pull request merged**